src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -28,13 +32,26 @@ internal class NuGetPackageDownloader : INuGetPackageDownloader
         };
 
         private readonly IFilePermissionSetter _filePermissionSetter;
-        private readonly ILogger _logger;
+        
+        /// <summary>
+        /// In many commands we don't passing console logger to reduce the verbosity
```

Comment:
I don't understand this comment, can you clarify/ reword?

Reply:
"In many commands we don't passing console logger and pass NullLogger instead to reduce the verbosity"

Reply:
will that clarify? We pass null logger, and so there is mostly nothing printed to the screen for a lot of commands like list and install